README.md

```
@@ -99,8 +99,43 @@ transaction:
 }
 ```
 
-Each event is output individually. The log output prints each event to stdout
-using Go's standard `slog` logging library.
+Each event is output individually. The log output supports two formats:
+
+- **text** (default) — human-readable, one line per event:
+
+  ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)